src/Commands/DevelopCommand.php

```diff
@@ -11,15 +11,19 @@ class DevelopCommand extends Command
 
     public function handle()
     {
+        // PHP_OS_FAMILY on Windows is "Windows"
+        $phpBinPackageDir = config('nativephp.binary_package');
+        $nativeBinaryPath = $phpBinPackageDir . 'bin/' . (PHP_OS_FAMILY === 'Windows' ? 'win' : 'mac');
```

Comment:
I think it makes sense to move this logic into a separate trait, as we need it for all commands that make use of the PHP binary. And when we add Linux support, we then only need to update it in one place.